cmd/kots/cli/install.go

```diff
@@ -447,14 +448,14 @@ func InstallCmd() *cobra.Command {
 				log.FinishSpinner()
 
 				switch status {
-				case storetypes.VersionPendingPreflight:
+				case storetypes.VersionPendingPreflight, storetypes.VersionPending:
```

Comment:
Will this run when there are no preflight checks defined?

Reply:
It shouldn't, because after the `ValidateAutomatedInstall` function above, the status will either be `pending_config`, `deployed`, or `deploying` if there were no preflights defined, because the application will be automatically deployed in that case.

Reply:
Hmm, we do deploy the version in a goroutine, so there might be a race condition there: https://github.com/replicatedhq/kots/blob/main/pkg/version/version.go#L83

Reply:
This should help: https://github.com/replicatedhq/kots/pull/4126/commits/6b388ee6d7dca149d1ab2a867acf978451c4de75